go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/tektoncd/operator
 
-go 1.21
+go 1.22.0
```

Comment:
Any reason for `1.22.0` and not `1.22` ?

Reply:
it is automatically added by go mod tidy and toolchain go1.22.4. but removing it now

Reply:
https://github.com/tektoncd/plumbing/pull/2043 should also help not having to remove `toolchain` in the future 🤞🏼